### ISSUE TITLE
[Suggestion] Use Atkinson Hyperlegible Font - greater readability and legibility for low vision readers

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3677]
[Atkinson hyperlegible Font](https://brailleinstitute.org/freefont) focuses on letterform distinction to increase character recognition and improve readability. 

See: https://github.com/pulibrary/Rails-project-Liz/commit/2dc143c30d5541d0ff1207a9c0ca45880c0061c4

Deque University is using Atkinson.

**Comment by christinach** [comment]
CJK example: https://catalog.princeton.edu/catalog/99125570074106421
with the Atkinson: 
![Screenshot 2023-08-16 at 5 34 21 PM](https://github.com/pulibrary/orangelight/assets/9905193/b91b70ef-d8c9-43a0-9a6f-4f431d55831f)

**Comment by christinach** [comment]
Hindi language example: https://catalog.princeton.edu/catalog/99102083273506421
![Screenshot 2023-08-16 at 5 36 01 PM](https://github.com/pulibrary/orangelight/assets/9905193/02fa3701-0030-49e3-a295-79f9b03accbf)

**Comment by christinach** [comment]
Arabic language example: https://catalog.princeton.edu/catalog/9947826143506421
![Screenshot 2023-08-16 at 5 38 31 PM](https://github.com/pulibrary/orangelight/assets/9905193/8ad278ae-803c-42ea-af65-bdd987c6f792)

**Comment by christinach** [comment]
Work in progress https://github.com/pulibrary/orangelight/tree/i3677-atkinson-font

**Comment by christinach** [comment]
We discussed this ticket during our Orangelight meeting on 12/7/2023. @escowles will bring this up to DSSG. We will keep the branch deployed in catalog-qa at least for a week from now so that subject experts on different languages can review the new font and provide feedback.

**Comment by jacobxperez** [comment]
@christinach Atkinson Hyperlegible only has support for Latin script based languages and it does not support Greek language, I know because I am working right now on adding Greek language support to [Atkinson Hyperlegible Pro](https://github.com/jacobxperez/atkinson-hyperlegible-pro) you may find the progress report in issue [Missing: Greek Language Support](https://github.com/jacobxperez/atkinson-hyperlegible-pro/issues/38).
